packages/p2p/src/stores/general-store.js

```diff
@@ -519,16 +524,8 @@ export default class GeneralStore extends BaseStore {
                         p2p_advertiser_info: 1,
                         subscribe: 1,
                     },
-                    [this.updateAdvertiserInfo, response => sendbird_store.handleP2pAdvertiserInfo(response)]
-                ),
-                order_list_subscription: subscribeWS(
-                    {
-                        p2p_order_list: 1,
-                        subscribe: 1,
-                        offset: 0,
-                        limit: this.list_item_limit,
-                    },
-                    [this.setP2pOrderList]
+                    [this.updateAdvertiserInfo, response => sendbird_store.handleP2pAdvertiserInfo(response)],
+                    this.is_p2p_user
```

Comment:
only calls if the user has 'active' or 'temp_ban' statuses, else no need to call to reduce load for BE